src/core_modules/capture-core/metaDataMemoryStoreBuilders/common/factory/searchGroup/SearchGroupFactory.js

```diff
@@ -124,7 +124,7 @@ export class SearchGroupFactory {
     async _buildSection(searchGroupAttributes: Array<SearchAttribute>) {
         const section = new Section((o) => {
             o.id = Section.MAIN_SECTION_ID;
-            o.showContainer = false;
+            o.showContainer = true;
```

Comment:
Hey @henrikmv ,
Can you tell me what are the steps to reproduce the scenario you fixed by setting `showContainer` to `true`? I don't really understand why the searchForm component is affected by your changes. 
Thanks!

Reply:
Hi @simonadomnisoru, thank you for your question. 

To fix the horizontal view, I have removed a div-element from the render function in `D2Section.component.js.` Therefore, I also had to move the `data-test` prop (and also a className prop). Without an alternative place for the data-test prop so that it applies to both rendering with and without section in the D2section.component, the data test prop now only applies when we are rendering with sections. I have set `o.showConatiner` to be true so that the `if statement on line 78` of `D2Section.component.js` should become false. In this way, the data test prop is included also for the search form.

Reply:
I think @simonadomnisoru's suggestion will work, but let’s see if we can break this down and simplify it. It was hard to reason about the original code (before your changes) so a solid refactor is due. Suggestion:

1. `renderSection` function: The main part here is that we should render `D2SectionFields`. `sectionMetaData.showContainer` decides if `D2SectionFields` should be wrapped in a Section-component or not. D2SectionFields should be passed the same props, regardless of whether we wrap it or not. common pattern:
```
	const  sectionFields = (<D2SectionFields……/>);
	
	if (sectionMetadata.showContainer) {
		return (
			<Section ….. >
				{sectionFields}
			</Section>
	);
	
	return sectionFields;
```

2. In the main render function let’s re-add the div with a data-test (name: «d2-section») and the original className (the conditional `cointainerCustomForm`) (the class could potentially have been moved to another place, but it's fine to leave it here for now)

3. In the `D2SectionFields.component` wrap everything in a `div`. The div should have the data-test «d2-section-fields» and it should conditionally add a class that sets `display: ‘flex’` (and the others css styles @simonadomnisoru suggested) when `props.formHorizontal` is true.

The tests will have to be updated. Most of them can probably use data-test «d2-section».